mysql/datadog_checks/mysql/queries.py

```diff
@@ -39,6 +39,20 @@
 
 SQL_WORKER_THREADS = "SELECT THREAD_ID, NAME FROM performance_schema.threads WHERE NAME LIKE '%worker'"
 
+SQL_USERS_CONNECTED = """\
+SELECT
+    processlist_user,
+    processlist_db,
+    processlist_state,
+    COUNT(processlist_user) AS connections
```

Comment:
To avoid confusion, can you update this to `COUNT(*) AS connections` or better: `COUNT(1) AS connections`.

Although please double-check that these are functionally equivalent and I'm not missing the point of counting the user column.